openmdao/docs/theory_manual/setup_stack.rst

```diff
@@ -13,9 +13,19 @@ to inputs, converging solvers, and calculating derivatives. It also includes set
 communicators.
 
 Setup also performs some level of model checking, mainly for critical errors. More extensive model
-checking can be done by setting "check" to True when calling `setup`, or by using the :ref:`openmdao command
+checking can be done by setting "check" when calling `setup`, or by using the :ref:`openmdao command
```

Comment:
It looks like `check_config` is also a public method now on Problem...  should that be mentioned?

Reply:
Done.